src/components/settings/SettingsLMSTab/tests/BlackboardConfig.test.jsx

```diff
@@ -76,97 +94,145 @@ describe('<BlackboardConfig />', () => {
         existingData={existingConfigData}
       />,
     );
-    fireEvent.change(screen.getByLabelText('API Client ID/Blackboard Application Key'), {
-      target: { value: 'test1' },
-    });
-    fireEvent.change(screen.getByLabelText('API Client Secret/Application Secret'), {
-      target: { value: 'test2' },
-    });
     fireEvent.change(screen.getByLabelText('Display Name'), {
       target: { value: 'displayName' },
     });
     fireEvent.change(screen.getByLabelText('Blackboard Base URL'), {
       target: { value: 'https://www.test.com' },
     });
     expect(screen.getByText('Submit')).not.toBeDisabled();
-    fireEvent.click(screen.getByText('Submit'));
+    act(() => {
+      fireEvent.click(screen.getByText('Submit'));
+    });
 
     const expectedConfig = {
       active: false,
       blackboard_base_url: 'https://www.test.com',
-      client_id: 'test1',
-      client_secret: 'test2',
       display_name: 'displayName',
       enterprise_customer: enterpriseId,
     };
-    expect(LmsApiService.updateBlackboardConfig).toHaveBeenCalledWith(expectedConfig, 1);
+    expect(mockUpdateConfigApi).toHaveBeenCalledWith(expectedConfig, 1);
   });
-  test('it creates new configs on submit', () => {
+  test('it creates new configs on submit', async () => {
     render(
       <BlackboardConfig
         enterpriseCustomerUuid={enterpriseId}
         onClick={mockOnClick}
         existingData={noExistingData}
       />,
     );
-    fireEvent.change(screen.getByLabelText('API Client ID/Blackboard Application Key'), {
-      target: { value: 'test1' },
-    });
-    fireEvent.change(screen.getByLabelText('API Client Secret/Application Secret'), {
-      target: { value: 'test2' },
+    act(() => {
+      fireEvent.change(screen.getByLabelText('Display Name'), {
+        target: { value: 'displayName' },
+      });
+      fireEvent.change(screen.getByLabelText('Blackboard Base URL'), {
+        target: { value: 'https://www.test.com' },
+      });
+    });
+    expect(screen.getByText('Authorize')).not.toBeDisabled();
+    await act(async () => {
+      fireEvent.click(screen.getByText('Authorize'));
```

Comment:
I think using user-events here instead is better, also probably don't need act around the click